receiver/azureeventhubreceiver/config.go

```diff
@@ -31,6 +31,7 @@ type Config struct {
 	Partition               string        `mapstructure:"partition"`
 	Offset                  string        `mapstructure:"offset"`
 	StorageID               *component.ID `mapstructure:"storage"`
+	Encoding                string        `mapstructure:"encoding"`
```

Comment:
Can we make this an enum and validate it is either raw or "data"?

Reply:
Done.